pkg/podmounter/mppod/creator.go

```diff
@@ -10,9 +10,10 @@ import (
 
 // Labels populated on spawned Mountpoint Pods.
 const (
-	LabelVersion    = "s3.csi.aws.com/mountpoint-version"
-	LabelPodUID     = "s3.csi.aws.com/pod-uid"
-	LabelVolumeName = "s3.csi.aws.com/volume-name"
+	LabelVersion          = "s3.csi.aws.com/mountpoint-version"
+	LabelPodUID           = "s3.csi.aws.com/pod-uid"
+	LabelVolumeName       = "s3.csi.aws.com/volume-name"
+	LabelCSIDriverVersion = "s3.csi.aws.com/csi-driver-version"
```

Comment:
Would it make sense to be more like `mounted-by-csi-driver-version`? (bad name, but my thinking is that its a bit unclear right now)

Reply:
Done, thanks!